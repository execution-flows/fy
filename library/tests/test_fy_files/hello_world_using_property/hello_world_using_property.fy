flow HelloWorld:
    property greeting using constant

    def -> None:
        print(self._greeting)
