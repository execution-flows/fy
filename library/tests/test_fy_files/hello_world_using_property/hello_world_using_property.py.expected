from base.execution_flow_base import ExecutionFlowBase

from tests.test_fy_files.hello_world_using_property import Greeting_UsingConstant_PropertyMixin


class HelloWorld_Flow(
    # Property Mixins
    Greeting_UsingConstant_PropertyMixin,

    # Base
    ExecutionFlowBase[None]
):
    def __call__(self) -> None:
        print(self._greeting)
